### ISSUE TITLE
Broken on Mojo 6.41

### ISSUE THREAD

**Issue by plicease** [clustericious/Clustericious#38]
works on 6.40

```
t/app_proxy.t ......................... 1/10 Use of uninitialized value $cb in method lookup at /home/ollisg/.perlbrew/libs/perl-5.23.7tc@dev/lib/perl5/Mojo/UserAgent/Transactor.pm line 118.
Can't locate object method "" via package "Mojo::UserAgent::Transactor" at /home/ollisg/.perlbrew/libs/perl-5.23.7tc@dev/lib/perl5/Mojo/UserAgent/Transactor.pm line 118.
```

**Comment by plicease** [comment]
https://travis-ci.org/clustericious/Clustericious/jobs/103814529

**Comment by plicease** [comment]
fixed in 1.15
